src/interfaces/new-request-interface.ts

```diff
@@ -55,6 +55,7 @@ export interface NewRequestIF {
   nrOriginal: Partial<NameRequestI>
   nrRequestNameMap: RequestNameMapI[]
   nrRequiredModalVisible: boolean
+  origin_entity_type_cd: EntityType
```

Comment:
I add this to save the `from` entity type because the request type cannot be determined before we select one from the allowable alteration list. However, `from` entity type determines which should be shown at this list.